TESTING-CHECKLIST.md

```diff
@@ -0,0 +1,81 @@
+## Testing
+
+### Login
+
+- [ ] Logout
+- [ ] Login with wrong password fails
+- [ ] Login with correct password
+
+### Sync
+
+- [ ] Create new note appears in other device
+- [ ] Changes to new note sync to other device
+- [ ] Changes to new note sync _from_ other device
+- [ ] New tag immediately syncs to other device
+- [ ] New tag immediately syncs _from_ other device
+- [ ] Removed tag immediately syncs to other device
+- [ ] Removed tag immediately syncs _from_ other device
+- [ ] Note publishes with link
+- [ ] Note unpublishes
+- [ ] Note publish change syncs _from_ other device (visible with dialog open)
+- [ ] Markdown setting syncs _from_ other device
+- [ ] Preview mode disappears/reappears when receiving remote changes to markdown setting
+- [ ] Note pinning syncs immediately from either direction
+- [ ] Note pinning works regardless if clicking in list view or from note info (and syncs after the editor is opened or the app is resumed from being in the background)
+- [ ] Viewing history on one device leaves note unchanged on other device
+- [ ] Restoring history immediately syncs note from both directions
+- [ ] After disabling network connectivity and making changes, selecting Log Out triggers an `Unsynced Notes Detected` alert
+- [ ] After going back online, changes sync
+
+### Note editor
+
+- [ ] Can preview markdown with Preview tab (👁 button on landscape tablet)
+- [ ] Can flip to edit mode with Edit tab (👁 button on landscape tablet)
+- [ ] Can toggle sidebar (in landscape tablet)
+- [ ] Tapping the `Insert checklist` icon inserts a checklist
+- [ ] Typing `- [x]` creates a checked checklist item
+- [ ] Typing `- [ ]` created an unchecked checklist item
+- [ ] Typing `-` creates a list
+- [ ] All list bullet types render to markdown lists
+- [ ] Added URL is linkified
+- [ ] Tapping on link opens it in browser
```

Comment:
```suggestion
- [ ] Tapping on link shows Link bar with options to view in browser, copy, and share
```

Tapping a link doesn't open in a browser immediately.  It shows a ***Link*** app bar with **_View in Browser_**, **_Copy_**, and **_Share_** options.  I'm not sure if we need to get into that level of detail with the checklist, but I want to point it out in case it was important to be specific.

Reply:
Ah, I missed that detail when I was updating the checklist. Thanks!